lib/payouts/services/verifySignature.js

```diff
@@ -4,16 +4,18 @@ const verifySignature = (body, signature, clientSecret) => {
   if (!(body && signature && clientSecret)) {
     throw Error(
       'Invalid Parameters: Please give request body,' +
-        'signature sent in X-Cf-Signature header and ' +
-        'clientSecret from dashboard as parameters',
+      'signature sent in X-Cf-Signature header and ' +
+      'clientSecret from dashboard as parameters',
     );
   }
 
-  const expectedSignature = crypto
-    .createHmac('sha256', clientSecret)
-    .update(body.toString())
-    .digest('hex');
+  var keys = Object.keys(body);
```

Comment:
@shahrukh-mansuri Can you please elaborate, the problem and how your code fixes it? 
The PR lacks details on what and how this fixes the issue.